shell/src/main/java/alluxio/shell/command/HelpCommand.java

```diff
@@ -31,6 +32,26 @@
  */
 @ThreadSafe
 public final class HelpCommand extends AbstractShellCommand {
+  private static final HelpFormatter HELP_FORMATTER = new HelpFormatter();
+
+  /**
+   * Prints the info about a command to the given print writer.
+   *
+   * @param command command to print info
+   * @param pw where to print the info
+   */
+  public static void printCommandInfo(ShellCommand command, PrintWriter pw) {
```

Comment:
@jsimsa not quite sure how to unit test this rather than hard-coding the output.
In this method, a lot of formatting work is hidden by using  `org.apache.commons.cli.HelpFormatter` to create the formatted (e.g., wrapped, padded output). I could not easily assert substring in the output.

Suggestion?